spec/controler_spec.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+
+require 'controler'
+require 'user_interface'
+
+RSpec.describe Controler do
+  let(:actions) { [double('NullAction')] }
+  let(:output) { StringIO.new }
+  let(:valid_input) { StringIO.new('1') }
+
+  it 'tells the user_interface to print the menu' do
+    ui = UserInterface.new(valid_input, output)
+    ctrl = Controler.new(ui, actions)
+
+    allow(actions[0]).to receive(:run)
+
+    ctrl.start
+
+    expect(output.string).to include(MENU_MESSAGE)
```

Comment:
Prefer to use the constants directly rather than redefine them in two places, that's two places to make changes to!
```ruby
expect(output.string).to include(UserInterface::MENU_MESSAGE)
```